dotcom-rendering/src/web/components/EmailSignup.tsx

```diff
@@ -85,13 +88,14 @@ export const EmailSignup = ({
 					Sign up to <span>{name}</span> today
 				</p>
 				<div css={noHeightFromTabletStyles}>
```

Comment:
Really minor point - we should probably update the comment on line 55 that still refers to a `NewsletterFrequency` component.